examples/cpp/csm/csm_greeter_server.cc

```diff
@@ -0,0 +1,126 @@
+/*
+ *
```

Comment:
let's just use single line comment for copyright as well

Reply:
Ack